### ISSUE TITLE
Remember Successful or Unsuccessful Combinations

### ISSUE THREAD

**Issue by Kanegasi** [TLDRMissions/TLDRMissions#236]
**TLDR Version of your idea**
TLDR version: Have the simulator remember combinations it already tried on a mission.

**Slightly longer version**
In order to avoid really long simulations, have the simulator remember really bad results so that they don't simulate that combination again, or remember successful combinations, or something as simple as remembering how many simulations a mission accumulates and put that at the front of the list when you have the "Hardest" mission option enabled so that you still have a decent selection of followers to get a successful combination. Basically, some form of machine learning where the amount of simulations get lower the more you use the addon.

**Additional context**

**Comment by teelolws** [comment]
It has been doing this for a while. You can see the difference yourself if you don't sent out a mission, and test it again.

There are two cases where the "results cache" becomes ineffective and gets cleared out:

1. When the addon has a major update. These include changes to the Simulator Engine, and its no good relying on cached results that are now unreliable
2. If your follower heals. Followers that are low health heal about 1HP every 2 minutes.
